contracts/GradualDeliveryCrowdsale.sol

```diff
@@ -27,21 +27,12 @@ import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
  * several times when the contract owner calls deliverTokenRatio() method.
  */
 contract GradualDeliveryCrowdsale is Crowdsale, Ownable {
+    using SafeMath for uint;
```

Comment:
[`uint` and `uint256` look the same type](http://solidity.readthedocs.io/en/v0.4.21/types.html#integers).  did you have to declare it separately?

Reply:
Yes, indeed they are the same.  OpenZeppelin seems to use `uint` like `size_t` in C/C++, so I followed the style, though I am not sure why they do so.